frontend/ui/translator.lua

```diff
@@ -277,6 +277,16 @@ This is useful:
                     G_reader_settings:flipNilOrTrue("translator_from_auto_detect")
                 end,
             },
+            {
+                text = _("Show romanizations"),
+                help_text = _("Displays source language text in latin characters. This is useful for reading languages with non-latin scripts."),
+                checked_func = function()
+                    return G_reader_settings:isTrue("translator_from_romanizations")
+                end,
+                callback = function()
+                    G_reader_settings:flipTrue("translator_from_romanizations")
+                end,
+            },
             {
                 text_func = function()
                     local lang = G_reader_settings:readSetting("translator_from_language")
```

Comment:
(May be you could name the setting `translator_with_romanizations` or `translator_show_romanizations` to avoivd "same prefix but different purpose" with `translator_from_language` with using `from`?)

Reply:
Yeah, I re-used the `_from_` segment to mean that the romanizations come from the "from" language (rather than the "to" language).

But maybe that's a little confusing.

Reply:
Renamed to `translator_with_romanizations`